game/scripts/npc/abilities/spirit_breaker_nether_strike.txt

```diff
@@ -43,7 +43,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage"                                          "150 200 250 500 750"
+        "damage"                                          "150 200 250 1000 1750"
```

Comment:
WTF is this? Lmao

Reply:
Pocket Money's PR's are worse than mine lmao @DarkoniusXNG

Reply:
Work in progress boys, trust